src/EditorFeatures/Core.Wpf/StringCopyPaste/WpfStringCopyPasteService.cs

```diff
@@ -40,9 +45,16 @@ public bool TrySetClipboardData(string key, string data)
 
                 copy.SetData(GetFormat(key), data);
 
-                Clipboard.SetDataObject(copy);
+                // Similar to what WinForms does, except that instead of blocking for up to 1s, we only block for up to 250ms.
+                // https://github.com/dotnet/winforms/blob/0f76e65878b1a0958175f17c4360b8198f8b36ba/src/System.Windows.Forms/src/System/Windows/Forms/Clipboard.cs#L31
+                Clipboard.SetDataObject(copy, copy: false, retryTimes: 5, retryDelay: 50);
                 return true;
             }
+            catch (ExternalException ex) when ((uint)ex.ErrorCode == CLIPBRD_E_CANT_OPEN)
```

Comment:
Emm.. is the goal of this change to be able to 'eat' this type of exception? 
What kind of exception does wpf gives us?

Reply:
Yes. That's the goal. The clipboard may be locked. If we can't access it, we just swallow the exception as 'nothing else we can do here'